src/V3Inline.cpp

```diff
@@ -283,7 +283,7 @@ class InlineRelinkVisitor final : public VNVisitor {
             // user2p is either a const or a var.
             FileLine* const flp = nodep->fileline();
             AstConst* const exprconstp = VN_CAST(nodep->user2p(), Const);
-            const AstVarRef* const exprvarrefp = VN_CAST(nodep->user2p(), VarRef);
+            AstVarRef* const exprvarrefp = VN_CAST(nodep->user2p()->cloneTree(false), VarRef);
```

Comment:
I think you are leaking the clone now, in the first `if (exprconstp)` breanch below, as it is not added to the tree in that case.

Reply:
Yes, you're right. I will fix it.